tests/common/mod.rs

```diff
@@ -4,7 +4,7 @@ pub const ZKEVM_LLVM: &str = "zkevm-llvm";
 pub const PACKAGE_VERSION: &str = env!("CARGO_PKG_VERSION");
 pub const ERA_LLVM_REPO_URL: &str = "https://github.com/matter-labs/era-compiler-llvm";
 pub const ERA_LLVM_REPO_TEST_BRANCH: &str = "v1.4.2";
-pub const ERA_LLVM_REPO_TEST_REF: &str = "714cc6aa05ea03dab3e4e504cb8c25973a33ea94";
+pub const ERA_LLVM_REPO_TEST_REF: &str = "714cc6aa0";
```

Comment:
Can we use the full commit hash here?
It's easy to get it via `git log` and the length of these short refs is really unpredictable...

Reply:
Yeah, fixed the tests code.